packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -35,7 +36,7 @@ async function createServer(
     const requirementsData = await requirements.resolveRequirements();
     const uberJar = path.resolve(
       extensionContext.extensionPath,
-      'out',
+      'dist',
```

Comment:
hum...this is probably also an issue when running int tests (requires bundling before execution).  I'll see if I can bake it into the bundle build.